performance-timeline/po-resource-single.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>PerformanceObservers: resource</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="performanceobservers.js"></script>
+<h1>PerformanceObservers: resource</h1>
+<p>
+New resources will <a href="https://w3c.github.io/performance-timeline/#dfn-queue-a-performanceentry">queue a PerformanceEntry</a>.
+</p>
+<div id="log"></div>
+<script>
+  async_test(function (t) {
+    var img_location = document.location.origin
+       + "/performance-timeline/resources/square.png";
+    var observer = new PerformanceObserver(
+        t.step_func(function (entryList, obs) {
+          checkEntries(entryList.getEntriesByType("resource"),
+            [{ entryType: "resource", name: img_location}]);
+          observer.disconnect();
+          t.done();
+        })
+      );
+    observer.observe({entryTypes: ["resource"]});
+    var img = document.createElement("img");
+    img.src = img_location;
+    document.body.appendChild(img);
+    img = document.createElement("img");
+    img.src = img_location;
+    document.body.appendChild(img);
+  }, "Only one resource entry per resource");
```

Comment:
This one is a bit nuanced. Even changing the .src on the element to the same URL could have a performance optimization to avoid a network request and not have a resource entry. The exact same .src is being set on an added element BUT it is a different element. An ideal implementation of a browser in this test case would re-use existing resources. In general, I believe that it would be ok for the browser to re-use memory inline and to skip the resource timing in that scenario. But.. as Ilya hints, the image is scrolled out... then added again later, it MAY add a resource timing entry.

Does Web Platform Tests have the concept of an experimental test?
(I'm actually very curious about how this type of testing would play out as multiple browsers implement Resource Timing on Performance Observer but I'm not yet sure we've specced how this should behave.)